src/controllers/product/index.js

```diff
@@ -0,0 +1,7 @@
+import InternalError from '~/src/errors/internal-error'
+import { product } from '~/src/models'
+
+export const getProducts = () => product.findAll()
+  .catch(() => {
```

Comment:
I'd suggest catching errors at a higher level so that we keep our controllers lean.

Reply:
I've created a handler for this, please give me your point of view of that